.github/workflows/build.yml

```diff
@@ -105,7 +105,8 @@ jobs:
         config:
           # Don't forget to add a download-artifact steps in test and publish
         - '{ "tfm": "net6.0", "emsdk": "2.0.23" }' # eol: 2024-11-12
-        - '{ "tfm": "net7.0", "emsdk": "3.1.12" }' # eol: 2023-05-08
+        - '{ "tfm": "net7.0", "emsdk": "3.1.12" }' # eol: 2024-05-14
+        - '{ "tfm": "net8.0", "emsdk": "3.1.34" }' # eol: 2026-11-14
```

Comment:
I'm following up with the team to make sure this is still the right approach, and to see how likely it is that this number will change between now an GA.